apps/site/components/EOL/UnknownSeveritySection/index.tsx

```diff
@@ -0,0 +1,33 @@
+import { useTranslations } from 'next-intl';
+import type { FC } from 'react';
+
+import VulnerabilitiesTable from '#site/components/EOL/VulnerabilitiesTable';
+import type { UnknownSeverityVulnerability } from '#site/types/vulnerabilities';
+
+const UnknownSeveritySection: FC<{
+  vulnerabilities: Array<UnknownSeverityVulnerability>;
+  hasKnownVulnerabilities: boolean;
+}> = ({ vulnerabilities, hasKnownVulnerabilities }) => {
+  const t = useTranslations();
+
+  if (!vulnerabilities.length) {
```

Comment:
Since this is already checking length to conditional render why can't VulnerabilitiesTable do the same? This is an inconsistent application of design patterns tho 🤔